src/System.Text.RegularExpressions/tests/RegexMatchTests0.cs

```diff
@@ -9,183 +9,75 @@
 public partial class RegexMatchTests
 {
     /*
-    Tested Methods:
-
-        public static Match Match(string input, string pattern);     Testing \B special character escape
-            "adfadsfSUCCESSadsfadsf", ".*\\B(SUCCESS)\\B.*"
-
-        public static Match Match(string input, string pattern);     Testing octal sequence matches
-            "011", "\\060(\\061)?\\061"
-
-        public static Match Match(string input, string pattern);     Testing hexadecimal sequence matches
-            "012", "(\\x30\\x31\\x32)"
-
-        public static Match Match(string input, string pattern);     Testing control character escapes???
-            "2", "(\u0032)"
-
+    public static Match Match(string input, string pattern);
+        - Testing \B special character escape: "adfadsfSUCCESSadsfadsf", ".*\\B(SUCCESS)\\B.*"
+        - Testing octal sequence matches: "011", "\\060(\\061)?\\061"
+        - Testing hexadecimal sequence matches: "012", "(\\x30\\x31\\x32)"
+        - Testing control character escapes???: "2", "(\u0032)"
     */
```

Comment:
I think we can remove these "Tested methods" comments across the board, unless you think they add value.